### PR TITLE
Nuget MyGet Fix

### PR DESCRIPTION
**Pull request by cschuchardt88**
#### Looks like `--disable-buffering` flag fixed it for me on my `myget`. Also fixed `Neo.ConsoleService` the version suffix wasnt getting appended.

image

### CODE REVIEW

**Comment by cschuchardt88** [comment]
@shargon I tested this my `myget` and it worked so, hopefully this fixes it.

File: src/Neo.ConsoleService/Neo.ConsoleService.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>1.2.0</Version>
+    <PackageId>Neo.ConsoleService</PackageId>
```
Comment (shargon) [inline src/Neo.ConsoleService/Neo.ConsoleService.csproj:4]: this change is required?

Comment (cschuchardt88) [reply src/Neo.ConsoleService/Neo.ConsoleService.csproj:4]: No, unless you want the same version to be push each time and also its doesnt append the suffix to the version. That's my reasoning. Just let me know what you want to do.

**Comment by shargon** [comment]
Can we test it first in dev-pack? I think that it should be easier to check it there and we only push one package

**Comment by cschuchardt88** [comment]
Sure I'll update the workflow on that repo.

**Review by shargon — APPROVED** [review]

**Pull request merged**